packages/docusaurus-theme-common/package.json

```diff
@@ -42,7 +42,7 @@
     "@types/react-router-config": "*",
     "clsx": "^1.2.1",
     "parse-numeric-range": "^1.3.0",
-    "prism-react-renderer": "^1.3.5",
+    "prism-react-renderer": "^2.0.6",
```

Comment:
If we upgrade react-live, we should probably allow 2.0.5 too to avoid duplicate deps, since react-live fixed the version to 2.0.5 in https://github.com/FormidableLabs/react-live/commit/50c2e28b2fcea1e183b6d6d23c2c29c18314d27c 

(cc @carloskelly13 wondering why)

Reply:
Glad to see the new exported types have cleaned this up a bit. Let me know if you have any others qs regarding PRR.

Reply:
@carloskelly13 is there a specific reason react-live is pinning 2.0.5, rather than `^2.0.5`?

Reply:
Good shout, I'll have a fix here to unpin that → https://github.com/FormidableLabs/react-live/pull/370

Reply:
@harryzcy `react-live@4.1.4` [published](https://www.npmjs.com/package/react-live/v/4.1.4) with the non-pinned version, now starting at `prism-react-rendere@^2.0.6`

Reply:
@harryzcy Good shout, I published a new version that fixes the pinned version → https://www.npmjs.com/package/react-live/v/4.1.5

This should now hit PRR `"prism-react-renderer": "^2.0.6",`